modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/FileEntryVideoEmbeddableHTMLItemSelectorReturnTypeResolver.java

```diff
@@ -43,16 +43,25 @@ public Class<FileEntry> getModelClass() {
 	public String getValue(FileEntry fileEntry, ThemeDisplay themeDisplay)
 		throws PortalException {
 
+		String html = null;
```

Comment:
can we inline this inside the json?

Reply:
Source formatter complains when I inline inside the json. It says to use an if else for better readability.

Reply:
??

I was suggesting more like

```
"html",
	() -> {
		if (((dlVideoRenderer != null) ...
		}

		return null;
	}
```

This way the SF is saying that is better with the else?

Reply:
You are right, that works.